cmd/cli/app/root.go

```diff
@@ -73,7 +73,7 @@ Configuration options include:
 - identity.cli.issuer_url
 - identity.cli.client_id
 
-By default, we look for the file as $PWD/config.yaml. You can specify a custom path via the --config flag, or by setting the MINDER_CONFIG environment variable.`,
+By default, we look for the file as $PWD/config.yaml and $HOME/.minder.yaml. You can specify a custom path via the --config flag, or by setting the MINDER_CONFIG environment variable.`,
```

Comment:
**suggestion (non-blocking):** follow XDG standards.

Personally, I'm a big fan of `$XDG_CONFIG_HOME`, we could use something like [xdg library](https://github.com/adrg/xdg) to help managing configuration on a per-OS basis. I'm fine either way, but I always appreciated projects that managed configuration in a standardized way.

Reply:
+1 to this suggestion